src/parser/parsers.h

```diff
@@ -1200,7 +1200,19 @@ template<typename Ctx> Result<> makeBreak(Ctx& ctx, Index pos) {
 }
 
 template<typename Ctx> Result<> makeBreakTable(Ctx& ctx, Index pos) {
-  return ctx.in.err("unimplemented instruction");
+  std::vector<typename Ctx::LabelIdxT> labels;
+  while (true) {
```

Comment:
I'd add a comment here explaining the logic and the intended outcome of parsing at least one label.